### ISSUE TITLE
[Bounty Creation Unit Test] Create mock data for bounty creation

### ISSUE THREAD

**Issue by elraphty** [stakwork/sphinx-tribes#996]
Create mock data for unit tests for bounty creation: 

- Create a mock for bounties array save to local storage [peopleBounties Mock](https://robertmarshall.dev/blog/how-to-mock-local-storage-in-jest-tests/). Look at [Local storage mock](https://github.com/stakwork/sphinx-tribes/pull/995) for mocking of localstorage
- Be able to add a bounty
- Be able to update a bounty
- And also be able to delete a bounty

Bounty interface

```
export interface PersonBounty {
  person?: any;
  body?: any;
  org_uuid?: any;
  title?: string;
  description?: string;
  owner_id: string;
  created?: number;
  show?: boolean;
  assignee?: any;
  wanted_type: string;
  type?: string;
  price?: string;
  codingLanguage: string;
  estimated_session_length: string;
  bounty_expires?: string;
  commitment_fee?: number;
}

```

The getPeopleBounties function in [store/main.ts](https://github.com/stakwork/sphinx-tribes/blob/4de7b2884b2870c6f53e7220e7b8c9645deaefb9/frontend/app/src/store/main.ts) makes an API query to get all bounties, write a test that mocks the get bounties endpoint when bounties are listed it should be persisted to the ```peopleBounties```
localStorage.

The saveBounty function in [store/main.ts](https://github.com/stakwork/sphinx-tribes/blob/4de7b2884b2870c6f53e7220e7b8c9645deaefb9/frontend/app/src/store/main.ts) makes an API query to add a new  bounty, write a test that mocks the save bounty endpoint when a new bounty is saved it should be added and persisted to the ```peopleBounties```
localStorage.

The getBountyById function in [store/main.ts](https://github.com/stakwork/sphinx-tribes/blob/4de7b2884b2870c6f53e7220e7b8c9645deaefb9/frontend/app/src/store/main.ts) makes an API query to get a  single bounty, write a test that mocks the get bounty by Id endpoint after a successful request the bounty, you should ascertain that the return bounty matches the bounty in peopleBounties array in local storage with the same bounty id.

The deleteBounty function in [store/main.ts](https://github.com/stakwork/sphinx-tribes/blob/4de7b2884b2870c6f53e7220e7b8c9645deaefb9/frontend/app/src/store/main.ts) makes an API query to delete a bounty, write a test that mocks the delete bounty endpoint after a successful delete the bounty should be removed from ```people bounties``
localStorage.

**Comment by Khaalid-oh** [comment]
Unit test

**Comment by ecurrencyhodler** [comment]
Closed by #1156
